File: mediapicker/src/main/java/iam/thevoid/mediapicker/rxmediapicker/metrics/Duration.java
package iam.thevoid.mediapicker.rxmediapicker.metrics;

import java.util.concurrent.TimeUnit;

public class Duration {
    private TimeUnit timeUnit;
    private long duration;

    public Duration(long duration, TimeUnit timeUnit) {
        this.timeUnit = timeUnit;
        this.duration = duration;
    }

    public long getSeconds() {
        return timeUnit.toSeconds(duration);
    }
}

File: mediapicker/src/main/java/iam/thevoid/mediapicker/picker/metrics/Duration.kt
package iam.thevoid.mediapicker.picker.metrics

import java.util.concurrent.TimeUnit

data class Duration @JvmOverloads constructor(
        private val duration: Long = -1,
        private val timeUnit: TimeUnit = TimeUnit.MILLISECONDS
) {

    val seconds: Long
        get() = timeUnit.toSeconds(duration)
}